ccm_web/server/appHandler.ts

```diff
@@ -41,15 +37,9 @@ class AppHandler {
       { host: db.host, dialect: 'postgres', logging: false }
     )
 
-    let devMode
     let staticPath
     const cookieOptions = { secure: true, sameSite: 'None' }
-    if (this.envOptions.isDev) {
-      // Setting devMode to true because cookies seem to be lost during proxy?
-      devMode = true
-    } else {
       staticPath = this.envOptions.staticPath
-    }
```

Comment:
You can probably pass `staticPath` directly into `Provider.setup` now. It looks like there are some unnecessary `let` uses or indentations.

Reply:
Done